docs/docs-content/clusters/public-cloud/azure/azure-cloud.md

```diff
@@ -21,121 +21,98 @@ Palette supports deploying and managing Kubernetes clusters in an Azure account.
 
 To register an Azure cloud account in the Palette console
 
-1. Log in to [Palette](https://console.spectrocloud.com).
+1. Log in to [Palette](https://console.spectrocloud.com) as a tenant admin.
 
```

Comment:
Let's shorten the title "Enable Azure Cloud Account Registration to Palette", it's a bit too long and we don't need the word Palette.